insalan/tournament/models.py

```diff
@@ -615,4 +640,27 @@ def clean(self):
                 _("Utilisateur⋅rice déjà inscrit⋅e dans un tournoi de cet évènement")
             )
 
+class Caster(models.Model):
+    
+    pseudo = models.CharField(
+        max_length=42,
+        null=False,
+        blank=False,
+        verbose_name=_("Pseudo du casteur")
+    )
```

Comment:
I feel like "pseudo" still sound very French

Reply:
What would you suggest to replace ?

Reply:
"username" or "in_game_name" maybe ?

Reply:
Not for casters, simply something like "name"?

Reply:
"name" sounds good to me.